docs/integrations-in-rancher/istio/istio.md

```diff
@@ -47,6 +47,10 @@ Kiali is a comprehensive visualization aid used for graphing traffic flow throug
 
 You can check the health of the service mesh, or drill down to see the incoming and outgoing requests to a single component.
 
+:::note
+If you are attempting to generate a session token to log into Kiali, note that for the service account name the Rancher deployment uses `kiali` instead of `kiali-service-account` when retrieving the session token.
```

Comment:
Based on this [comment](https://github.com/rancher/rancher-docs/issues/1340#issuecomment-2200933034) on the original issue, which points to [these Kiali docs](https://kiali.io/docs/faq/authentication/), and a similar code snippet presented as an [example](https://kiali.io/docs/installation/installation-guide/example-install/#access-the-kiali-server-ui) under the Kiali install docs, it's a little ambiguous if Kiali expects a service account named `kiali-service-account`, or if the `kiali-service-account` in the code snippet is intended to be a placeholder for an arbitrary name for the account. 

Since I'm not sure if it was intended as a placeholder, this suggestion doesn't mention `kiali-service-account`. 

```suggestion
The name of the Kiali service account in Rancher is `kiali`. Use this name if you are writing commands that require you to enter the name of the Kiali service account—for example, if you are trying to generate or retrieve a session token.
```

Reply:
Hey Marty thanks for the review! I added your suggestion as well as a Kiali docs link in the above text section and a link to the Kiali token authentication FAQ page in the note section. Let me know what you think!